liubai-backends/liubai-laf/cloud-functions/ai-entrance.ts

```
@@ -392,9 +392,12 @@ function mapBots(
 /** check out if it's a directive, like "召唤..." */
 class AiDirective {
 
+  private static _bots: AiBot[] = []
+
   static check(
     entry: AiEntry
   ): AiDirectiveCheckRes | undefined {
+    this._bots = AiHelper.getAvailableBots()
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)